### DISCUSSION TITLE
Website description in Google

### DISCUSSION THREAD

**Discussion by cdmcdermott** [adityatelange/hugo-PaperMod#447] [Q&A]
Where do I specify a website description for google to display when people search. I have configured the below in config.yml

params:
       description: <description here is ignored>

profileMode:
        enabled: true 
        title: <this title is picked up by google but not yahoo, duck duck go>
        subtitle: <this title is picked up by google but not yahoo, duck duck go>

When i search for my site in google it appears that google just picks up all text on the front page e.g. title, subtitle, tags, categories.
How do i specify I detailed description for SEO

**Comment by adityatelange** [comment]
All params are mentioned here. 

https://github.com/adityatelange/hugo-PaperMod/wiki/Variables

**Reply by cdmcdermott** [reply]
Thanks. It looks like I have set the correct parameters: title, description, profileMode.title and profileMode.subtitle.

I am still not convinced search engines are utilising these parameters correctly as DuckDuckGo and Yahoo show raw HTML and google seems to just scrape all text on the front page.

Is this how the theme is meant to work or a possible bug?

Many thanks for an awesome theme by the way

**Reply by adityatelange** [reply]
I don't think so it is a bug. 
It takes time for search engines to crawl websites and duckduckgo and bing does it a lot slower than google.

**Reply by ariascosb** [reply]
In my case duckduckgo and bing show it correctly, but google does not
